src/GovUk.Education.ExploreEducationStatistics.Admin/Startup.cs

```diff
@@ -407,10 +407,10 @@ public virtual void ConfigureServices(IServiceCollection services)
 
             services.Configure<PreReleaseOptions>(Configuration);
             services.Configure<LocationsOptions>(Configuration.GetSection(LocationsOptions.Locations));
-            services.Configure<ReleaseApprovalOptions>(
-                Configuration.GetSection(ReleaseApprovalOptions.ReleaseApproval));
+            services.Configure<ReleaseApprovalOptions>(Configuration.GetSection(ReleaseApprovalOptions.ReleaseApproval));
```

Comment:
As a team I think we've been using line wrapping set up at 120 characters which is why this is formatted the way it is, so I think you should revert this change and set your line wrapping  at 120.

EES-4595 will enforce this standard when it comes in.